[maint] pytave CI failures from mercurial certifcates error

@mtmiller, FYI: this is happening:
```
$ if [ "x$PYTAVE" = "xyes" ]; then hg clone https://bitbucket.org/mtmiller/pytave pytave; pushd pytave; pwd; autoreconf --install || exit 1; ./configure PYTHON_VERSION=$PYTHON_VERSION && make || exit 1; popd; fi
warning: connecting to bitbucket.org using legacy security technology (TLS 1.0); see https://mercurial-scm.org/wiki/SecureConnections for more info
(using CA certificates from /etc/ssl/certs/ca-certificates.crt; if you see this message, your Mercurial install is not properly configured; see https://mercurial-scm.org/wiki/SecureConnections for how to configure Mercurial to avoid this message)
abort: error: _ssl.c:510: error:1408F10B:SSL routines:SSL3_GET_RECORD:wrong version number
/home/travis/.travis/job_stages: line 104: pushd: pytave: No such file or directory
/home/travis/build/cbm755/octsympy
autoreconf: 'configure.ac' or 'configure.in' is required
```
source: https://travis-ci.org/cbm755/octsympy/jobs/482257504

This isn't urgent: Pytave tests are optional and are currently broken (pending https://bitbucket.org/mtmiller/pytave/issues/90/char-pysysversion-does-not-return-a-string).

--- a/.travis.yml
+++ b/.travis.yml
@@ -5,19 +5,19 @@ language: generic
 
 matrix:
   include:
-    - env: PYTHON_VERSION=2 PYTAVE=no  SYMPY_VER=1.2   OCT=ppa   DOCTEST=yes COLUMNS=80
-    - env: PYTHON_VERSION=3 PYTAVE=no  SYMPY_VER=1.2   OCT=ppa   DOCTEST=yes COLUMNS=80
-    - env: PYTHON_VERSION=2 PYTAVE=no  SYMPY_VER=1.3   OCT=ppa   DOCTEST=yes COLUMNS=80
-    - env: PYTHON_VERSION=3 PYTAVE=no  SYMPY_VER=1.3   OCT=ppa   DOCTEST=yes COLUMNS=80
-    - env: PYTHON_VERSION=2 PYTAVE=no  SYMPY_VER=1.3   OCT=daily DOCTEST=no  COLUMNS=80
-    - env: PYTHON_VERSION=3 PYTAVE=no  SYMPY_VER=1.3   OCT=daily DOCTEST=no  COLUMNS=80
-    - env: PYTHON_VERSION=2 PYTAVE=yes SYMPY_VER=1.3   OCT=daily DOCTEST=no  COLUMNS=80
-    - env: PYTHON_VERSION=3 PYTAVE=yes SYMPY_VER=1.3   OCT=daily DOCTEST=no  COLUMNS=80
+    - env: PYTHON_VERSION=2 PYTHONIC=no  SYMPY_VER=1.2   OCT=ppa   DOCTEST=yes COLUMNS=80
+    - env: PYTHON_VERSION=3 PYTHONIC=no  SYMPY_VER=1.2   OCT=ppa   DOCTEST=yes COLUMNS=80
+    - env: PYTHON_VERSION=2 PYTHONIC=no  SYMPY_VER=1.3   OCT=ppa   DOCTEST=yes COLUMNS=80
+    - env: PYTHON_VERSION=3 PYTHONIC=no  SYMPY_VER=1.3   OCT=ppa   DOCTEST=yes COLUMNS=80
+    - env: PYTHON_VERSION=2 PYTHONIC=no  SYMPY_VER=1.3   OCT=daily DOCTEST=no  COLUMNS=80
+    - env: PYTHON_VERSION=3 PYTHONIC=no  SYMPY_VER=1.3   OCT=daily DOCTEST=no  COLUMNS=80
+    - env: PYTHON_VERSION=2 PYTHONIC=yes SYMPY_VER=1.3   OCT=daily DOCTEST=no  COLUMNS=80
+    - env: PYTHON_VERSION=3 PYTHONIC=yes SYMPY_VER=1.3   OCT=daily DOCTEST=no  COLUMNS=80
   allow_failures:
-    - env: PYTHON_VERSION=2 PYTAVE=no  SYMPY_VER=1.3   OCT=daily DOCTEST=no  COLUMNS=80
-    - env: PYTHON_VERSION=3 PYTAVE=no  SYMPY_VER=1.3   OCT=daily DOCTEST=no  COLUMNS=80
-    - env: PYTHON_VERSION=2 PYTAVE=yes SYMPY_VER=1.3   OCT=daily DOCTEST=no  COLUMNS=80
-    - env: PYTHON_VERSION=3 PYTAVE=yes SYMPY_VER=1.3   OCT=daily DOCTEST=no  COLUMNS=80
+    - env: PYTHON_VERSION=2 PYTHONIC=no  SYMPY_VER=1.3   OCT=daily DOCTEST=no  COLUMNS=80
+    - env: PYTHON_VERSION=3 PYTHONIC=no  SYMPY_VER=1.3   OCT=daily DOCTEST=no  COLUMNS=80
+    - env: PYTHON_VERSION=2 PYTHONIC=yes SYMPY_VER=1.3   OCT=daily DOCTEST=no  COLUMNS=80
+    - env: PYTHON_VERSION=3 PYTHONIC=yes SYMPY_VER=1.3   OCT=daily DOCTEST=no  COLUMNS=80
 
 before_install:
   - if [ "x$OCT" = "xppa" ]; then
@@ -35,10 +35,10 @@ before_install:
         sudo apt-get install -y python$PYTHON_VERSION-dev python$PYTHON_VERSION-pip;
     fi
 # we can drop numpy after https://bitbucket.org/mtmiller/pytave/issues/84 is resolved.
-  - if [ "x$PYTAVE" = "xyes" ]; then
+  - if [ "x$PYTHONIC" = "xyes" ]; then
         pip$PYTHON_VERSION install --user numpy;
     fi
-  - if [ "x$PYTAVE" = "xyes" ]; then
+  - if [ "x$PYTHONIC" = "xyes" ]; then
         sudo apt-get install -y liboctave-dev;
     fi
 
@@ -50,9 +50,9 @@ install:
         wget https://github.com/catch22/octave-doctest/releases/download/v0.6.1/doctest-0.6.1.tar.gz;
         octave -W --no-gui --eval "pkg install doctest-0.6.1.tar.gz";
     fi
-  - if [ "x$PYTAVE" = "xyes" ]; then
-        hg clone https://bitbucket.org/mtmiller/pytave pytave;
-        pushd pytave;
+  - if [ "x$PYTHONIC" = "xyes" ]; then
+        git clone https://gitlab.com/mtmiller/octave-pythonic.git pythonic;
+        pushd pythonic;
         pwd;
         autoreconf --install || exit 1;
         ./configure PYTHON_VERSION=$PYTHON_VERSION && make || exit 1;
@@ -67,15 +67,15 @@ script:
   - stty cols $COLUMNS rows 40
   - tput cols; stty size
   - pushd inst
-  - if [ "x$PYTAVE" = "xyes" ]; then
+  - if [ "x$PYTHONIC" = "xyes" ]; then
         export IPC=native;
     else
         export IPC=default;
     fi
-  - octave --path=$TRAVIS_BUILD_DIR/pytave -W --no-gui --eval "sympref ipc $IPC; r = octsympy_tests; exit(r)";
+  - octave --path=$TRAVIS_BUILD_DIR/pythonic -W --no-gui --eval "sympref ipc $IPC; r = octsympy_tests; exit(r)";
   - cat fntests.log
   - if [ "x$DOCTEST" = "xyes" ]; then
-        octave --path=$TRAVIS_BUILD_DIR/pytave -W --no-gui --eval "pkg load doctest; sympref ipc $IPC; syms x; r = doctest('.'); exit(~r)";
+        octave --path=$TRAVIS_BUILD_DIR/pythonic -W --no-gui --eval "pkg load doctest; sympref ipc $IPC; syms x; r = doctest('.'); exit(~r)";
     fi
   - popd
 

